learning/readme.md

```diff
@@ -1,6 +1,25 @@
-# Learning (Engineering roles)
+# Learning
 
-## Onboarding
+## Learn Tech
+
+Every Friday between 13:30 and 17:00 the team come together to focus on Core Skills.
+
+Learn Tech consists of a number of tracks, where the whole company learn everything we need to know to make Made Tech an even more brilliant team. Tracks will run periodically throughout the year, between which more free form learn tech sessions or personal development will take place.
+
+* All permanent employees are expected to attend and be on time.
+* The time should be focussed on attaining, or be defining and/or improving Core Skills.
+
+## Show & Tell
```

Comment:
I _think_ we more commonly call these "Showcases" internally, even though "Show & Tell" is a common name across gov.